.tekton/network-observability-operator-bundle-push.yaml

```diff
@@ -30,6 +30,10 @@ spec:
     value: '{{revision}}'
   - name: output-image
     value: quay.io/redhat-user-workloads/ocp-network-observab-tenant/netobserv-operator/network-observability-operator-bundle:{{revision}}
+  - name: image-expires-after
+    value: 14d
```

Comment:
@OlivierCazade I'm adding that because we generate a lot of builds and it surely is going to fill the quay repo :-/ 
idk if 14d is ok - I guess the point is that the last build has to be still there when it's time to release. If necessary maybe we can increase to 30d.